src/udev/meson.build

```diff
@@ -53,6 +53,28 @@ if conf.get('HAVE_ACL') == 1
         libudevd_core_sources += ['udev-builtin-uaccess.c']
 endif
 
+if conf.get('HAVE_STAP') == 1
+        probes_h = custom_target(
+                'probes.h',
+                input : 'probes.d',
+                output : 'probes.h',
+                command : ['dtrace', '-C', '-h', '-s', '@INPUT@', '-o', '@OUTPUT@'])
+
+        probes_o = custom_target(
+                'probes.o',
+                input : 'probes.d',
+                output : 'probes.o',
+                command : ['dtrace', '-C', '-G', '-s', '@INPUT@', '-o', '@OUTPUT@'])
+
+        probes_a = custom_target(
+                'probes.a',
+                input : probes_o,
+                output : 'probes.a',
+                command : ['ar', 'rcs', '@OUTPUT@', '@INPUT@'])
```

Comment:
you are basically building a static lib with this, right? isn't the right approach for that to use meson's native static lib concepts? (i have no idea, just asking)

Reply:
Yeah, this should be done using `static_library`.

Reply:
Again, not relevant after changes after @t-8ch.